library/cc/engine_builder.cc

```diff
@@ -100,6 +100,10 @@ std::string EngineBuilder::generateConfigStr() {
       {"dns_preresolve_hostnames", this->dns_preresolve_hostnames_},
       {"dns_refresh_rate", fmt::format("{}s", this->dns_refresh_seconds_)},
       {"dns_query_timeout", fmt::format("{}s", this->dns_query_timeout_seconds_)},
+      {"dns_resolver_name", "envoy.network.dns_resolver.cares"},
+      {"dns_resolver_config",
+       "{\"@type\":\"type.googleapis.com/"
```

Comment:
nit: I'd suggest a string quoting approach that doesn't require a bunch of inline escaping